plugins/woocommerce-blocks/assets/js/data/checkout/actions.ts

```diff
@@ -7,7 +7,6 @@ import { AdditionalValues } from '@woocommerce/settings';
  * Internal dependencies
  */
 import { ACTION_TYPES as types } from './action-types';
-import { ReturnOrGeneratorYieldUnion } from '../mapped-types';
 
 // `Thunks are functions that can be dispatched, similar to actions creators
```

Comment:
```suggestion
// Thunks are functions that can be dispatched, similar to actions creators
```

Unrelated to your PR, but this backtick here looks accidental. There's another one in https://github.com/woocommerce/woocommerce/blob/785f1a482bd2ce437ed771cff24275c6613b9ace/plugins/woocommerce-blocks/assets/js/data/payment/actions.ts#L17.